.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Title
+[Provide a succinct and descriptive title for the pull request]
```

Comment:
I think title is redundant.

In general this project is small enough and this template would be overwhelming